components/about/TwitterTestimonials.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div
+    class="mx-auto tweet-widget"
```

Comment:
nit: `mx-auto` could be moved to css

Reply:
Could be, however, I'd argue that one of the things that using Vuetify gives us is convenience classes...